### PR TITLE
IAB Data Deletion Request Framework

### PR DESCRIPTION
**Pull request by max-ostapenko**
Newly introduced framework for programmatic [data deletion requests](https://iabtechlab.com/standards/data-deletion-request-framework/)

---

**Test websites**:

- https://data-deletion-request-framework.glitch.me/

### CODE REVIEW

File: dist/privacy.js [hunk 1]
```diff
@@ -25,7 +25,73 @@ function testPropertyStringInResponseBodies(pattern) {
...
+   * @param {function} parser - The function to parse the response.
+   * @returns {Promise<Object>} The parsed response or an error object.
+   */
+const fetchAndParse = async (url, parser) => {
+  const timeout = 5000;
+  /*
+  Google's sellers.json size is 120Mb as of May 2024 - too big for custom metrics.
+  It's available at realtimebidding.google.com/sellers.json, so not part of crawled pages list.
+  More details: https://support.google.com/authorizedbuyers/answer/9895942
+  */
```
Comment (tunetheweb) [inline dist/privacy.js:39]: Is this needed? Or just copied from elsewhere?

**Review by tunetheweb — APPROVED** [review]
LGTM with one question.

File: dist/privacy.js (hunk 1 above)
Comment (max-ostapenko) [reply dist/privacy.js:39]: It's required here to clarify that the top ad tech player is not covered by this metric. And this bias should be considered during analysis.

Comment (max-ostapenko) [reply dist/privacy.js:39]: We added this comment when implementing it in May 2024.

Comment (tunetheweb) [reply dist/privacy.js:39]: Yes I understand that for ads.js. Is it the same for the re-use of this function in privacy.js? Is `/dsrdelete.json` also 120MB?

Comment (max-ostapenko) [reply dist/privacy.js]: Sure, you're right. It's just copy-pasted comment.

Really need to check out the build tool asap.

**Pull request merged**